### ISSUE TITLE
Define variables with X-Control

### ISSUE THREAD

**Issue by MulteBerry** [ldrolez/clyphx-live10#43]
Just a quick question that I think I know the answer to (sadly) but just wanted to double check...

I've got a setup where a midi input has to define some variables before performing an action (as this action depends on the variable). I've tried making the midi trigger an x-clip that defines the variable but the x-clip's action is performed after the x-control even though it's last in the action list (probably due to the ≈100ms delay). So

**Is it possible to define variables with a MIDI controller via X-Controls?**
Or could there maybe be another way of approaching this?

**Comment by ldrolez** [comment]
Yes you can define variables in X-Controls. But you have more details, or a minimal live set to share?
